modules/nf-core/minimap2/align/meta.yml

```diff
@@ -25,6 +25,11 @@ input:
       description: |
         List of input FASTA or FASTQ files of size 1 and 2 for single-end
         and paired-end data, respectively.
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing reference information
```

Comment:
I wonder if it's worth mentioning it's not used for anything

Reply:
Is it mentioned in other modules ? I don't remember seeing it.

That said, I think it could be misleading. Yes, there is no written occurrence of `meta2` in the module, but it doesn't prevent it from still being used, for instance in a closure for `task.ext.args`, `task.memory`, etc (which is the way I intend to use it).

Reply:
No not always. I seem to remember that the original modules from Harshil didn't include an entry for each one.

However you make a good point, so fine as it is. My approval stands as before :+1: